include/swift/AST/SourceFile.h

```diff
@@ -734,9 +734,11 @@ struct DenseMapInfo<swift::SourceFile::ImportedModuleDesc> {
                               StringRefDMI::getTombstoneKey());
   }
   static inline unsigned getHashValue(const ImportedModuleDesc &import) {
-    return combineHashValue(ImportedModuleDMI::getHashValue(import.module),
-           combineHashValue(ImportOptionsDMI::getHashValue(import.importOptions),
-                            StringRefDMI::getHashValue(import.filename)));
+    return detail::combineHashValue(
```

Comment:
I'm not sure if functions from the `detail` namespace are supposed to be called here.

Reply:
Scratch that, I don't think there is a better way. LLVM is indeed only using it as an implementation detail, but Swift had its own copy which is now deleted.